### ISSUE TITLE
[Broken Website] msn.com not working properly

### ISSUE THREAD

**Issue by lbickle** [darkreader/darkreader#9812]
# Broken Website Report

## Website Address

www.msn.com
## Steps To Reproduce

Tested on Edge and Firefox. In Edge 
 - browse to the page and it initially is completely white as seen in Figure A
 - Clicking the dark reader ext shows that it is on correctly (dynamic/auto) 
 - After clicking "on" it changes to what looks like Figure B
 - Switch back to Auto will leave the site looking like Figure B until you close the browser and/or tab and go back to it after
   which it reverts back to figure A  
## Screenshots

![Fig A](https://user-images.githubusercontent.com/113444442/189909632-0b3ce11f-2fc2-4065-b415-97c4cced8d2d.jpg)
![Fig B](https://user-images.githubusercontent.com/113444442/189910589-aa857f43-0c92-419d-b78f-4153a976d2c3.jpg)
![Dark Setting](https://user-images.githubusercontent.com/113444442/189910616-89f66d10-752f-4d12-91a0-163cd8523058.jpg)

## System Information

- OS:  Windows 11 Pro 21H2
- Browser:  Microsoft Edge 105.0.1343.33 (Official build) (64-bit)
- Dark Reader Version:  4.9.57

**Comment by lbickle** [comment]
Anything else that is needed for this? Just want to make sure I have provided everything needed.

**Comment by Gusted** [comment]
Hi @lbickle!

Sorry for the no response, everything is good from your side. But I hadn't much info/progress on this as msn.com is using some *funky* techniques that Dark Reader isn't handling well, but it is also handling it partially. So somewhere in the process Dark Reader is blown, but that's currently hard to pin point.

**Comment by lbickle** [comment]
Yeah, definitely odd as it worked fine for the longest time and in some cases, you could clear cache on the browser, and it would "fix" it for a while but after whatever they did last it seems to not work right at all now. If there is anything I can do on this end to assist or provide/test anything by all means just let me know and I am happy to do so.

**Comment by Gusted** [comment]
I'm able to reproduce it on my own, so need to assist me anymore. Just needs a lot of time and debugging to get this figured out, instances of this like virustotal.com(#4682) is quite hard to debug as they rely on "modern web technologies", which can be quirky sometimes and have "funny behavior".
